### ISSUE TITLE
Source files for /static

### ISSUE THREAD

**Issue by privatejava** [kalcaddle/kodbox#333]
Hi team
Thanks for this wonderful software. I was wondering if there is a way to customize the UI on my own using UI codes. But all i can see was minified js files under /static. Are the UI elements all closed source ?

Thanks

**Comment by kalcaddle** [comment]
You can use plugins to expand the front-end and back-end functions. The front-end js/css is the packaged code.

You can perform expansion processing by monitoring the mount point time. For details, please refer to plugins such as drawio.

**Comment by privatejava** [comment]
Yes I am more interested in customizing the front-end  js/css which is in packaged code . Can we get the actual source code for that packaged code ?

**Comment by kalcaddle** [comment]
For reference, you can view the source code of the front-end and back-end of the program;

https://demo.kodcloud.com/#s/90zCPzDQ
